FULL_HELP_DOCS.md

```diff
@@ -480,7 +480,7 @@ stellar contract invoke ... -- --help
 ###### **Options:**
 
 * `--id <CONTRACT_ID>` — Contract ID to invoke
-* `--is-view` — View the result simulating and do not sign and submit transaction
+* `--is-view` — View the result simulating and do not sign and submit transaction. Ieprecated use `--send=no`
```

Comment:
sp: `Deprecated`

Reply:
Yeah this is why we shouldn't check in this file. I updated the source code where these comments come from by accepting a suggestion from Leigh, but then I need to re generate this file locally...

Reply:
```suggestion
* `--is-view` — View the result simulating and do not sign and submit transaction. Deprecated use `--send=no`
```

Reply:
Git history shows it being fixed in the code, then being changed back to `Ieprecated` in the code. Automation won't prevent us from undoing typo fixes in the code. I don't think we need more automation for this.

I'm going to push back on introducing more complexity for the sake of having a simply accessible help file that we can review in PRs as a way to understanding the UX/UI clearly.

Reply:
Fair enough.